modules/dnn/src/onnx/onnx_importer.cpp

```diff
@@ -427,6 +429,32 @@ void ONNXImporter::addLayer(LayerParams& layerParams,
     }
 }
 
+void ONNXImporter::expandMid(const LayerParams& layerParams, opencv_onnx::NodeProto& node_proto,
+                             const std::string& input, size_t n, int axis)
+{
+    String base_name = layerParams.name + "/copy_";
+    std::vector<std::string> input_names;
+    for (int j = 0; j < n; j++)
+    {
+        std::ostringstream ss;
+        ss << j;
+        LayerParams copyLP;
+        copyLP.name = base_name + ss.str();
```

Comment:
Use `format("%s/copy_%d", layerParams.name, j) - it will be faster and less code.

Reply:
done